User-Help/GlobalTrustLevels.md

```diff
@@ -0,0 +1,138 @@
+| **View private messages and annotations** | n/a                                                                                  | n/a                                                                                                  | n/a                                              | n/a                                                                                                 | n/a                                                                               | Full access                                                    |
+| **Access to all other moderation tools**  | n/a                                                                                  | n/a                                                                                                  | n/a                                              | n/a                                                                                                 | n/a                                                                               | Full access                                                    |
+
+
+
+# Glossary of terms
+
+### Prerequisites
+
+Minimum activity required to reach a given trust level.
+
+### Posts
+
+Typically posts are questions. Depending on the category, however, they can also include other formats like blogs or wikis.
+
+### Responses 
+
+A response is a standalone reply to a post, typically a fully researched answer to a question. It may be something else depending on category, e.g. a submission to a contest. 
+
+### Comments
+
+A comment, in comparison to a response, is a short note referring to either a post or comment. It is usually used for clarification or to request more information. 
+
+### Flags
+
+Flags send a user contribution to a moderation queue for review. Use these to report material inappropriate for a particular site, or anything that breaks the Code of Conduct or Terms of Service. 
+
+### Upvotes
+
+Upvotes help a post or answer gain visibility and act as an indicator of how the community values the contribution. They are also used in combination with downvotes to determine if it is well-received. 
+
+### Suggest edits
+
+Sometimes a user's contribution could use a little help. This could be due to grammar and formatting issues, or maybe the original poster has mentioned important information in a comment and needs to add it to the original post. Suggesting edits helps improve the quality of a site's content.
+
+### Suggest duplicates
+
+You may not have been the first person to ask this question! Duplicates aren't necessarily a bad thing, but for people looking for information it's helpful to have links to any previous posts on the same topic. 
```

Comment:
Let's make that second part a little more positive, something like: "When that happens, adding a link to the earlier question helps you see answers more quickly."

Reply:
Is there also a "suggest related post" action?

Reply:
No, for that people can leave comments.  We might in the future automatically show all linked posts in some central place, like in the right column perhaps.

The reason we have "suggest duplicate" is that it calls for action from the author, either to confirm it's a dupe or to edit to better explain the difference.  So we have an action for that.  Related posts are just informational, so we don't need an explicit action for that.